qdrant_client/context_headers.py

```
@@ -0,0 +1,43 @@
+from contextlib import asynccontextmanager, contextmanager
+from contextvars import ContextVar
+from typing import AsyncIterator, Iterator
+
+_context_headers: ContextVar[dict[str, str]] = ContextVar("_context_headers", default={})
```

Ruff (0.15.5)

[warning] Do not use mutable data structures for `ContextVar` defaults

(B039)